README.md edits

lasersPew opened the pull request.

NamanShergill commented:
> I just changed the readme. Te things I omitted is included btw in #51

Yep I went through it! I mostly mentioned the cherry pick command because GitHub thinks every file is being changed, so you can only cherry pick the Readme, or I can manually move the changes over with my comments I mentioned above if you would prefer that?

NamanShergill commented:
Oh whoops, this completely slipped my mind, sorry! Gonna work on adding the wiki asap now that I'm able to work on the project again.

HikaruIchijyo commented:
> Oh whoops, this completely slipped my mind, sorry! Gonna work on adding the wiki asap now that I'm able to work on the project again.

Yay finally

lasersPew commented:
Reopened this PR as I added more changes to the README

- Brought back the features
- Changed Logo Link
- Retractable Screenshots
- Added Buy me a Coffee image

NamanShergill commented:
Hey! Thanks for fixing the issue with all files being marked as changed (that would have messed git blame and would have made it hard to track where exactly a specific part of code was changed in the past).

There are a couple of changes I would like to mention (for example, files under assets are compiled with the app, so might need to move the coffee icon), but I’m currently on vacation so will share them in the next weekend. 

Thanks for the PR, the repo did need a better  organised readme

lasersPew committed:
Changed README

Added Logo link
Added Buy me a Coffee link
Added link to wiki
Added Retractable Screenshots

lasersPew commented:
> There are a couple of changes I would like to mention (for example, files under assets are compiled with the app, so might need to move the coffee icon), but I’m currently on vacation so will share them in the next weekend.

Would placing it onto ./repoassets do the trick?

lasersPew committed:
Changed icon Size

RKerkhof123 reviewed (changes requested):
assets/bmac.svg
